### PR TITLE
[VTA] Fix TSIM compile error in Linux (add missing -fPIC flag)

### PR DESCRIPTION
**Pull request by liangfu**
This PR fixed the compilation issue raised in #3797 , which requires running `make` command twice to compile the generated .cpp files into the shared library. In addition, the missing `-fPIC` flag has also been added for fixing the compile error in Linux;

@vegaluisjose Please review.

### CODE REVIEW

**Comment by tmoreau89** [comment]
Thanks for the fixes @liangfu !

File: vta/hardware/chisel/Makefile [hunk 1]
```diff
@@ -108,13 +107,16 @@ default: lib
...
+	$(CXX) -fPIC $(cxx_flags) -c $^ -o $@
 
 $(verilator_build_dir)/tsim_device.o: tsim_device.cc
-	$(CXX) -c $(cxx_flags) $^ -o $@
+	$(CXX) -fPIC $(cxx_flags) -c $^ -o $@
 
 $(lib_path): $(verilator_build_dir)/V$(TOP_TEST).cpp $(cxx_objs)
-	$(CXX) $(cxx_flags) $(ld_flags) $(cxx_objs) -o $@
+	for f in $(shell find $(verilator_build_dir)/*.cpp); do \
+    $(CXX) -fPIC $(cxx_flags) -c $${f} -o $${f}.o ; \
```
Comment (tmoreau89) [inline vta/hardware/chisel/Makefile]: fix spaces/tabs

**Comment by tmoreau89** [comment]
I've left a comment to fix indentation/tabs; otherwise it all looks good!

File: vta/hardware/chisel/Makefile (hunk 1 above)
Comment (liangfu) [reply vta/hardware/chisel/Makefile]: The indentation now looks better.

**Pull request merged**